src/eventHandlers/AccountMetadataEmittedEvent/handlers/handleEcosystemMainAccountMetadata.ts

```diff
@@ -148,6 +148,14 @@ async function upsertEcosystemMainAccount({
         ? metadata.isVisible
         : true,
     lastProcessedVersion: makeVersion(blockNumber, logIndex).toString(),
+    color:
+      'color' in metadata
+        ? metadata.color
+        : unreachableError('Missing color in Ecosystem Main Account metadata'),
+    avatar:
```

Comment:
I see that the `type` is always emoji, but should it be handled here anyways e.g. for future changes or consistency?

Reply:
We don't store the type either way. In the DB it's always a plain string. At the API, you get back the `__typename`. It's the same logic as in projects, if I understand correctly your point.